targets/switch/p4src/openflow.p4

```diff
@@ -80,7 +96,7 @@ action nop () {
 }
 
 action terminate_cpu_packet() {
-    modify_field(standard_metadata.egress_spec,fabric_header.dstPortOrGroup);
+    modify_field(standard_metadata.egress_port,fabric_header.dstPortOrGroup);
```

Comment:
Egress spec is valid for ingress pipeline while egress port is valid for egress pipeline in p4lang. So this should remain "egress spec."